requirements/dev.txt

```diff
@@ -43,7 +43,7 @@ idna==3.4
     # via requests
 imagesize==1.4.1
     # via sphinx
-importlib-metadata==5.2.0
+importlib-metadata==6.0.0
```

Comment:
No major breaking change in the new version https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v600